.fossa.yml

```diff
@@ -0,0 +1,14 @@
+# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
+# Visit https://fossa.com to learn more
+
+version: 2
+cli:
+  server: https://app.fossa.com
+  fetcher: custom
+  project: reaper-client-go
+analyze:
+  modules:
+  - name: github.com/k8ssandra/reaper-client-go/reaper
```

Comment:
double check the target modules:
`github.com/k8ssandra/reaper-client-go/reaper`

Reply:
This looks consistent with fossa documentation.